ehr2meds/make_synthetic_data.md

```
@@ -0,0 +1,82 @@
+  # Synthetic data workflow
```

markdownlint-cli2 (0.22.0)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)